source/external-interface-design/external-interface-design.rst

```diff
@@ -139,11 +139,7 @@ The Chirpstack is comprised of the following 5 modules.
 
 3. `Gateway bridge <https://www.chirpstack.io/gateway-bridge/>`__
 
-4. `Geolocation
```

Comment:
If the old Geolocation service is unused in OS2iot (which I believe it is) this is fine.

Reply:
The separate Geolocation server has been deprecated and is now just included directly in the Network Server